functions.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var numOfGallons = function(MPG,distance){
+    var gallons = distance/MPG;
+    console.log(gallons);
```

Comment:
Don't output - not part of the function's responsibility to output, makes it less useful.
Just output the value instead.